repository/g.json

```diff
@@ -915,6 +915,17 @@
 				}
 			]
 		},
+		{
+			"name": "GLanguage",
+			"details": "https://github.com/JoseCarlosSkar/sublime-glanguage",
+			"labels": ["language syntax", "snippets"],
+			"releases": [
+				{
+					"sublime_text": "*",
```

Comment:
> '.sublime-syntax' support has been added in build 3092 and there is no '.tmLanguage' fallback file

Would be nice to have a fix-it hint. Change the line above to `"sublime_text": ">=3092",`

Or rather, assume everyone is on the latest 3211.